hil/client/network.py

```diff
@@ -15,6 +15,17 @@ def list(self):
             url = self.object_url('networks')
             return self.check_response(self.httpClient.request("GET", url))
 
+        @check_reserved_chars()
+        def list_network_attachments(self, network, project):
+            """Lists nodes connected to a network"""
+            url = self.object_url('network', network, 'attachments')
+            if project == "all":
+                return self.check_response(self.httpClient.request("GET", url))
+            else:
+                params = {'project': project}
+                return self.check_response(
+                        self.httpClient.request("GET", url, params=params))
```

Comment:
@zenhack what's your opinion on handling a guard block? For example, when I see a code like this, I tend to get rid of the `else:` since that would require less indentation and 1 less line. Just curious what the best practice is.

Reply:
good question. curious too.

Reply:
Yeah, that's my general inclination.